operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -86,6 +86,16 @@ externalConfigSecretName: '{{ include "mimir.resourceName" (dict "ctx" . "compon
 # -- When 'useExternalConfig' is true, then changing 'externalConfigVersion' triggers restart of services - otherwise changes to the configuration cause a restart.
 externalConfigVersion: '0'
 
+# --Vault Agent config to mount secrets to TLS configurable components. This requires Vault and Vault Agent to already be running.
+vaultAgent:
+  enabled: false
+  roleName: ""
+  clientCertPath: ""
```

Comment:
Please add docstring to explain what this is? How do I know what to write here?
Will there be an associated doc written as well?